atecc608a/atecc508a_config_dev.h

```diff
@@ -148,7 +148,11 @@ const uint8_t template_config_508a_dev[]  =
   0x13, 0x00, /* 104-105 KeyConfig slot 4 */
   0x13, 0x00, /* 106-107 KeyConfig slot 5 */
   0x13, 0x00, /* 108-109 KeyConfig slot 6 */
-  0x13, 0x00, /* 110-111 KeyConfig slot 7 */
+  /* Slot 7 is used to contain ECDH master secret, so according to the KeyType description,
+   * KeyType has to be 0b111.
+   * 0x1F00 - 0001 1111 0000 0000, the only difference being:
+   * (4-2) 111 KeyType - Not an ECC key, but "any other kind of data, key, or secret". */
```

Comment:
There are more 1 bits set than just the KeyType. Could you please add a comment for the other bits, as you've done with other entries in this table?

Reply:
Both set and unset bits have a meaning here, and to be fair for all of them - I'd have to describe them all. In this case, both the set and unset ones that are unchanged fall under the "same as before" category, which is indicated by `the only difference being`.  It's the same policy as used in line 78.

Reply:
OK, so we are re-using the previous comment.

```
   * (1)     1      PubInfo - Public version of a key can always be generated.
   * (0)     1      Private - Contains a private key
```